src/software/thunderscope/thunderscope_config.py

```diff
@@ -217,6 +219,7 @@ def configure_base_fullsystem(
             in_window=True,
             anchor="Referee Info",
             position="below",
+            stretch=WidgetStretchData(y=4),
```

Comment:
I am not quite sure why we need this? Is it to make Thunderscope look like full screen again?

Reply:
It should make the bottom widgets take less space on the screen relative to the field widget, on launch. Currently without this change they take half of the screen. Thunderscope should still launch full screen.

Reply:
Also could you try to launch thunderscope a couple of times with these changes, and a few times on master, and see if you notice a difference in launch time?